pkgs/os-specific/darwin/apple-source-releases/libsbuf/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  bootstrapStdenv,
+  fetchurl,
+  meson,
+  ninja,
+  stdenv,
+}:
+
+# Apple ships libsbuf with macOS 14 but does not provide any source releases.
+# Fortunately, it’s a single file library that can be made to build on Darwin using the source from FreeBSD.
+bootstrapStdenv.mkDerivation (finalAttrs: {
+  pname = "libsbuf";
+  version = "14.1.0";
+
+  outputs = [
+    "out"
+    "dev"
+    "man"
+  ];
+
+  srcs = [
```

Comment:
Things in the Darwin stdenv bootstrap can’t use `fetchgit` with filters, and I’d rather not fetch a multi-GiB repo for a handful of files. Also, because it’s in the bootstrap, update scripts basically don’t work (because they depend on curl, which is also prohibited in the bootstrap). I agree it would be nice if there were one, but I don’t know what a good approach would be.